tasks/bundle.yml

```diff
@@ -7,12 +7,19 @@
     changed_when: redmine_ruby_setup.stdout != ''
 
   - name: Bundle - Ensure bundler is installed.
-    command: bash -lc "gem install bundler"
+    command: bash -lc "gem install bundler {% if redmine_bundler_version and redmine_bundler_version != "" %}-v {{ redmine_bundler_version }}{% endif %}"
     args:
       chdir: "{{ redmine_home }}"
     register: redmine_bundler_install
     changed_when: redmine_bundler_install.stdout.find('Installing') != -1
 
+  - name: Bundle - Ensure bundler is updated.
```

Comment:
I think the update of gem should be before the bundler install step.

Reply:
C'est corrigé